src/internal/serviceenv/config.go

```diff
@@ -19,6 +19,7 @@ type GlobalConfiguration struct {
 	S3GatewayPort       uint16 `env:"S3GATEWAY_PORT,default=600"`
 	PPSEtcdPrefix       string `env:"PPS_ETCD_PREFIX,default=pachyderm_pps"`
 	Namespace           string `env:"PACH_NAMESPACE,default=default"`
+	PipelineLabel       string `env:"PACH_WORKER_LABEL"`
```

Comment:
Apologies if I just missed it, but I couldn't figure out right away: how does this get set?

Reply:
This is only set by the testing framework, to specify which test pachd each worker belongs to. It's not really used in production at all.